.travis.yml

```diff
@@ -76,3 +76,8 @@ matrix:
     - language: d
       install: dmd parser_d.d
       script: ./test_parser.sh ./parser_d
+
+    - language: elixir
+      script:
+        - mix escript.build
```

Comment:
don't you need to cd into the project's directory?

Reply:
WIP I need to do this even before script so that mix can be executed, and I am discovering pitfalls all the way long, but thx for your help :blush:

I will also try to get a modern Elixir version, 1.0.2 is probably not cutting it with the `jason`.

So this will take some time :sob: